charts/netdata/README.md

```diff
@@ -80,107 +80,104 @@ deletes the release.
-| `sd.image.tag`                           | Service-discovery image tag                                                                                                                            | Latest stable release (e.g. `v0.1.0`)                                     |
-| `sd.image.pullPolicy`                    | Service-discovery image pull policy                                                                                                                    | `Always`                                                                  |
-| `sd.child.enabled`                       | Add service-discovery sidecar container to the netdata child pod definition                                                                            | `true`                                                                    |
-| `sd.child.resources`                     | Child service-discovery container CPU/Memory resource requests/limits                                                                                  | `{resources: {limits: {cpu: 50m, memory: 150Mi}, requests: {cpu: 50m, memory: 100Mi}}}`                                                                      |
-| `sd.child.configmap.name`                | Child service-discovery ConfigMap name                                                                                                                 | `netdata-child-sd-config-map`                                             |
-| `sd.child.configmap.key`                 | Child service-discovery ConfigMap key                                                                                                                  | `config.yml`                                                              |
-| `sd.child.configmap.from.file`           | File to use for child service-discovery configuration generation                                                                                       | `sdconfig/sd-child.yml`                                                   |
-| `sd.child.configmap.from.value`          | Value to use for child service-discovery configuration generation                                                                                      | `{}`                                                                      |
+| Parameter                                | Description                                                                                                                                            | Default                                                                                 |
+|------------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------------|
```

Comment:
The table is broken, see https://github.com/netdata/helmchart/blob/knats-remove-wgetImage/charts/netdata/README.md#configuration

Reply:
Fixed